examples/third_party/pygwalker/example.py

```diff
@@ -0,0 +1,31 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "pygwalker==0.4.9.13",
+#     "vega-datasets==0.9.0",
+# ]
+# ///
+import marimo
+
+__generated_with = "0.9.14"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def __():
+    import pygwalker as pyg
```

Comment:
Are you able to achieve rendering the Pygwalker interface using this import? I tried it with datasets hosted through cloud links (S3), tried running this example as is too and it results in the class instance being printed:
`<pygwalker.api.pygwalker.PygWalker object at 0x00000124E362B470>`


I have found using these imports useful:
```
from pygwalker.api.marimo import walk # Usage -> walk(_df, spec="spec.json")
import pygwalker.api.marimo as pyg # Usage -> pyg.walk(_df)
```
this is with regards to the way it was implemented in the [Pygwalker PR](https://github.com/Kanaries/pygwalker/pull/647).